pandas/core/arrays/datetimes.py

```diff
@@ -1065,19 +1065,19 @@ def date(self):
 
         return tslib.ints_to_pydatetime(timestamps, box="date")
 
-    year = _field_accessor('year', 'Y', "\n The year of the datetime\n")
+    year = _field_accessor('year', 'Y', "\n The year of the datetime.\n")
     month = _field_accessor('month', 'M',
-                            "\n The month as January=1, December=12 \n")
-    day = _field_accessor('day', 'D', "\nThe days of the datetime\n")
-    hour = _field_accessor('hour', 'h', "\nThe hours of the datetime\n")
-    minute = _field_accessor('minute', 'm', "\nThe minutes of the datetime\n")
-    second = _field_accessor('second', 's', "\nThe seconds of the datetime\n")
+                            "\n The month as January=1, December=12. \n")
+    day = _field_accessor('day', 'D', "\nThe days of the datetime.\n")
+    hour = _field_accessor('hour', 'h', "\nThe hours of the datetime.\n")
+    minute = _field_accessor('minute', 'm', "\nThe minutes of the datetime.\n")
+    second = _field_accessor('second', 's', "\nThe seconds of the datetime.\n")
     microsecond = _field_accessor('microsecond', 'us',
-                                  "\nThe microseconds of the datetime\n")
+                                  "\nThe microseconds of the datetime.\n")
     nanosecond = _field_accessor('nanosecond', 'ns',
-                                 "\nThe nanoseconds of the datetime\n")
+                                 "\nThe nanoseconds of the datetime.\n")
     weekofyear = _field_accessor('weekofyear', 'woy',
-                                 "\nThe week ordinal of the year\n")
+                                 "\nThe week ordinal of the year.\n")
```

Comment:
In theory we removed all thos `\n` in #24099. Can you take a look at the PR and see if they were forgotten and if they can be removed here?

Reply:
I believe I got all of them, let me know if I missed any. Thanks.